draft-ietf-quic-transport.md

```diff
@@ -1151,22 +1140,39 @@ stateless_reset_token (0x0006):
 : The Stateless Reset Token is used in verifying a stateless reset, see
   {{stateless-reset}}.  This parameter is a sequence of 16 octets.
 
+A client MUST NOT include a stateless reset token.  A server MUST treat receipt
+of a stateless_reset_token transport parameter as a connection error of type
+TRANSPORT_PARAMETER_ERROR.
+
+An endpoint MAY use the following transport parameters:
+
+initial_max_stream_id_bidi (0x0002):
+
+: The initial maximum stream ID parameter contains the initial maximum stream
+  number the peer may initiate for bidirectional streams, encoded as an unsigned
+  32-bit integer.  This value MUST be a valid bidirectional stream ID for a
+  peer-initiated stream (that is, the two least significant bits are set to 0 by
+  a server and to 1 by a client).  If an invalid value is provided, the
+  recipient MUST generate a connection error of type TRANSPORT_PARAMETER_ERROR.
+  Setting this parameter is equivalent to sending a MAX_STREAM_ID
+  ({{frame-max-stream-id}}) immediately after completing the handshake.  The
+  maximum bidirectional stream ID is set to 0 if this parameter is absent,
+  preventing the creation of new bidirectional streams until a MAX_STREAM_ID
+  frame is sent.  Note that a default value of 0 does not prevent the
+  cryptographic handshake stream (that is, stream 0) to be used.
```

Comment:
This is perhaps a difference in usage, but I would have said "from being used" instead of "to be used".